docs/deployment/configuration.rst

```diff
@@ -66,8 +66,8 @@ these should be writen to the ``config.toml`` file or stored in
 environmental options in the service ini (if using 12-factor style).
 
 Dev:
-- ``--p2p.skip_upnp``: Don't try to determine external address.
-  Needed if running without internet (eg. in an airplane)
+- ``--p2p.upnp --proxy_app noop``: Don’t try to determine external address
```

Comment:
The command needs noop in single quotes ``--p2p.upnp --proxy_app 'noop'``: Don’t try to determine external address

Reply:
Why need the --proxy_app line at all for the skip upnp stuff?

I think default is now to not attempt, and `--p2p.upnp` enables it (or `--p2p.upnp=false` to explicitly disable if enabled via config of environment.

(I spent a lot of time with cobra, the flag parser they use)

Reply:
I added --proxy_app through trial and error when having a go at starting tendermint. This combo worked for me.